src/test/scala/is/hail/io/ImportPlinkSuite.scala

```diff
@@ -68,4 +68,37 @@ class ImportPlinkSuite extends SparkSuite {
   @Test def testPlinkImportRandom() {
     Spec.check()
   }
+
+  @Test def testA1Major() {
+    val a1major = hc.importPlinkBFile("src/test/resources/fastlmmTest", a2Major = false)
+      .annotateGenotypesExpr("g = Genotype(g.GT)")
+      .toVDS
+
+    val a2major = hc.importPlinkBFile("src/test/resources/fastlmmTest", a2Major = true)
```

Comment:
This fastlmmTest is synthetic with no missing genotypes and alleles 1 2 or 2 1 in the .bim. Do you understand why the alleles are numbers rather than strings? Do you think it's sufficient for this test, or should you export a vcf to plink and read that back in?